<?xml version="1.0"?>
<sdf version="1.6">
  <model name="unknown_weed">
    <static>false</static>
    <link name="link">
      <inertial>
        <pose>0 0 0.005 0 0 0</pose>
        <mass>0.1</mass>
        <inertia>
          <ixx>0.0000658333</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.0000658333</iyy>
          <iyz>0</iyz>
          <izz>0.000125</izz>
        </inertia>
      </inertial>
      <collision name="collision_leaves">
        <geometry>
          <mesh>
            <uri>model://unknown_weed/meshes/unknown_weed.dae</uri>
          </mesh>
        </geometry>
        <max_contacts>10</max_contacts>
        <surface>
          <contact>
            <ode />
          </contact>
          <bounce />
          <friction>
            <ode>
              <mu>10</mu>
              <mu2>10</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <collision name="collision_stem">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <cylinder>
            <radius>0.03</radius>
            <length>0.005</length>
          </cylinder>
        </geometry>
        <max_contacts>10</max_contacts>
        <surface>
          <contact>
            <ode />
          </contact>
          <bounce />
          <friction>
            <ode>
              <mu>10</mu>
              <mu2>10</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://unknown_weed/meshes/unknown_weed.dae</uri>
          </mesh>
        </geometry>
      </visual>
    </link>
  </model>
</sdf>